readme.md

```diff
@@ -1,13 +1,5 @@
 # Microsoft Azure IoT SDK for .NET
 
-## V2 clients release notice
```

Comment:
wondering if we should be more explicit to explain what are the recent plan changes wrt V2 SDK.